src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -120,16 +123,17 @@ def migration_update_func(cmd, client, resource_group_name, server_name, migrati
         if operationSpecified is True:
             raise MutuallyExclusiveArgumentError("Incorrect Usage: Can only specify one update operation.")
         operationSpecified = True
-        r = migration_show_func(cmd, client, resource_group_name, server_name, migration_name, "Default")
+        r = migration_show_func(cmd, client, resource_group_name, server_name, migration_name)
         if r.get("properties").get("migrationMode") == "Offline":
             raise BadRequestError("Cutover is not possible for migration {} if the migration_mode set to offline. The migration will cutover automatically".format(migration_name))
-        properties = json.dumps({"properties": {"triggerCutover": "true", "DBsToTriggerCutoverMigrationOn": cutover}})
+        properties = json.dumps({"properties": {"triggerCutover": "true", "DBsToTriggerCutoverMigrationOn": r.get("properties").get("dBsToMigrate")}})
```

Comment:
do we have any tests to validate these changes?

Reply:
If not can you add the same? For breaking change how will you ensure that the reffred Python SDK package has desired change?

Reply:
I'll change the existing tests and they should work as such.

Reply:
did you find any existing set of tests verifying this change?

Reply:
I checked this. We did not add tests for this since for online, cutover is done through a manual process of checking the porgress of the migration when the target has caught up with the source. Automation tests were not added for this.